.github/workflows/tests.yml

```diff
@@ -19,6 +19,8 @@ jobs:
         os: [ubuntu-latest, macos-latest, windows-latest]
     steps:
       - uses: actions/checkout@v2
+	    with:
```

Comment:
Is this indentation correct?

Reply:
Looks like I messed up the whitespace when I copy/pasted it